noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1422,49 +1808,28 @@
-                            LIMIT 1
-                        ) period_start_date
-                    FROM arkivstruktur.dokumentbeskrivelse d
-                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
-                    JOIN arkivstruktur.arkivdel s ON s.systemid = r._ref_arkivdel
-                    LEFT JOIN arkivstruktur.mappe f ON f._id = r._parent_id
-                    WHERE ( CAST(d.opprettetdato AS DATE) < period_start_date AND s.arkivdelstatus <> 'Overlappingsperiode' ) OR d.opprettetdato IS NULL OR period_start_date IS NULL;
-                ]]>
-            </warnings>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Document Object count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT
-                        r._ref_arkivdel series_system_id,
-                        COUNT(*) document_objects,
-                        (d._ref_dokumentbeskrivelse IS NOT NULL) in_document_description,
-                        (d._ref_dokumentbeskrivelse IS NULL) in_record
-                    FROM arkivstruktur.dokumentobjekt d
-                    LEFT JOIN arkivstruktur.dokumentbeskrivelse dd ON dd._id = d._parent_id
-                    LEFT JOIN arkivstruktur.registrering r ON r._id = d._parent_id
-                    GROUP BY series_system_id, in_document_description, in_record
-
-                    UNION ALL
-
-                    SELECT 'total', count(*), TRUE in_document_description, TRUE in_record
-                    FROM arkivstruktur.dokumentobjekt;
+                            FROM addml.property property
+                            JOIN addml.properties parent_properties ON property._parent_id = parent_properties._id
+                            JOIN addml.additionalelement parent_element ON parent_properties._parent_id = parent_element._id
+                            WHERE property.name = 'startDate' AND parent_element.name = 'archivalPeriod'
+                            LIMIT 1
+                        ) period_start_date
+                    FROM arkivstruktur.dokumentbeskrivelse d
+                    JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
+                    JOIN arkivstruktur.arkivdel s ON s.systemid = r._ref_arkivdel
+                    LEFT JOIN arkivstruktur.mappe f ON f._id = r._parent_id
+                    WHERE ( CAST(d.opprettetdato AS DATE) < period_start_date AND s.arkivdelstatus <> 'Overlappingsperiode' ) OR d.opprettetdato IS NULL OR period_start_date IS NULL;
                 ]]>
-            </info>
+            </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST16">
         <title>Document object checksums</title>
+        <description>
+            Tests whether the Document Object checksums specified in arkivstruktur
```

Comment:
`Tests whether the document object checksums specified in arkivstruktur.xml match the ones that the validator calculated using the SHA256 algorithm`